datastore/elastic_search/query.py

```diff
@@ -312,6 +314,12 @@ def full_text_query(connection, index_name, doc_type, entity_name, sentences, fu
     try:
         response = _run_es_search(connection, msearch=True, **kwargs)
         results = _parse_es_search_results(response.get("responses"))
+    except es_exceptions.NotFoundError as e:
+        raise DataStoreRequestException(f'NotFoundError in datastore query on index: {index_name}', engine='elasticsearch',
```

Comment:
E501 line too long (123 > 119 characters)

Reply:
resolved in latest commit